router/middlewares/JSONHandler.go

```diff
@@ -1,10 +0,0 @@
-package middlewares
```

Comment:
Remove this changes from commit "fix typo in Authentication" because it's not a typo.